mempool/errors.go

```diff
@@ -11,9 +11,6 @@ var ErrTxNotFound = errors.New("transaction not found in mempool")
 // ErrTxInCache is returned to the client if we saw tx earlier.
 var ErrTxInCache = errors.New("tx already exists in cache")
 
-// ErrLaneNotFound is returned to the client when a lane is not found.
-var ErrLaneNotFound = errors.New("lane not found in mempool")
```

Comment:
Why are we removing this?

Reply:
I think these are changes in the target branch.